### ISSUE TITLE
V3 Yeelight Bulb continuous search

### ISSUE THREAD

**Issue by nox10** [vieira/homebridge-yeelight-wifi#83]
Hello,

Every time after homebridge restart my V3 Yeelight Bulb cannot be discovered by this plugin until I explicitly disable and re-enable developer mode in Yeelight app

**Comment by tikhonp** [comment]
Yes, I also had the same problem and after re-enable developer mode it worked

**Comment by vieira** [comment]
Do you have any network configuration that would block the multicast broadcast between the lamps and homebrige on port `udp/1982`?
